lona/pytest.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import pytest
+
+from lona import LonaApp
+
+
+class LonaContext:
+    def __init__(self, client, app):
+        self.client = client
+        self.app = app
+
+    def get_url(self):
+        return str(self.client.make_url(''))
+
+
+@pytest.fixture()
+def lona_app_context(request, aiohttp_client):
+    async def setup_lona_app_context(setup_app):
+        # setup lona app
+        project_root = os.path.dirname(request.node.module.__file__)
+        lona_app = LonaApp(project_root)
```

Comment:
```suggestion
        lona_app = LonaApp(request.fspath)
```

Reply:
Ouh! Even better! I changed it that way, and made ``project_root`` configurable.

Reply:
Is `project_root` a good name?
As far as I understand it has to be a path to a file. But reading this name I think it should be a path to directory.